doc/source/tutorial/ndimage.rst

```diff
@@ -461,7 +461,7 @@ information).
   object). The :func:`generic_filter1d` function iterates over the
   lines of an array and calls :func:`function` at each line. The
   arguments that are passed to :func:`function` are one-dimensional
-  arrays of the :ctype:`tFloat64` type. The first contains the values
+  arrays of the :c:type:`tFloat64` type. The first contains the values
```

Comment:
I think it's an old `numarray` type:

https://heim.ifi.uio.no/~inf3330/scripting/doc/python/NumPy/numarray/node48.html